front/src/reducers/osrdconf/osrdConfCommon/__tests__/commonConfBuilder.ts

```diff
@@ -119,6 +119,35 @@ export default function commonConfBuilder() {
       },
     ],
 
+    buildFormattedPathElectrificationRanges: (): RangedValue[] => [
+      { begin: 0, end: 10, value: '1500V' },
+      { begin: 10, end: 20, value: '25000V' },
+      { begin: 20, end: 25, value: '1500V' },
+    ],
+
+    buildFormattedPowerRestrictionRanges: (): PowerRestrictionRange[] => [
+      { begin: 0, end: 10, value: 'NO_POWER_RESTRICTION' },
```

Comment:
Use the const defined in PowerRestrictionSelector
```suggestion
      { begin: 0, end: 10, value: NO_POWER_RESTRICTION },
```

Reply:
fixed